config/300-taskrun.yaml

```diff
@@ -26,20 +26,13 @@ spec:
     - tekton-pipelines
   scope: Namespaced
   additionalPrinterColumns:
-    - name: Type
-      type: string
-      description: The cron spec defining the interval a CronJob is run
-      JSONPath: .status.conditions[0].type
     - name: Status
       type: string
-      description: The number of jobs launched by the CronJob
-      JSONPath: .status.conditions[0].status
-    - name: StartTime
+      description: nothing
```

Comment:
something more explicit than nothing would be nice

Reply:
Sure, I will pick up some meaningful description, thanks